src/main/scala/org/globalforestwatch/layers/Layer.scala

```diff
@@ -184,9 +195,10 @@ trait RequiredILayer extends RequiredLayer with ILayer {
   /**
     * Define how to fetch data for required Integer rasters
     */
-  def fetchWindow(window: Extent): ITile = {
+  def fetchWindow(windowKey: SpatialKey, windowLayout: LayoutDefinition): ITile = {
+    val layoutSource = LayoutTileSource.spatial(source, windowLayout)
     val tile = source.synchronized {
-      source.read(window).get.tile.band(0)
+      layoutSource.read(windowKey).get.band(0)
```

Comment:
Codacy found an issue: [Usage of get on optional type.](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6292950)